product_pricelist_extended/models/product_pricelist.py

```diff
@@ -18,11 +18,11 @@ class ProductPricelist(models.Model):
     """
```

Comment:
Revert formatting changes to this file and add a comment to disable black formatting. It is important it continues to match upstream as closely as possible.